carmirror/batch.go

```diff
@@ -168,26 +184,39 @@ func NewBatchSendOrchestrator() *BatchSendOrchestrator {
 
 // Notify notifies the orchestrator of a session event.
 // Events lead to state transitions in the orchestrator.
+//
```

Comment:
Is this comment about a problem you still have, or is it solved by the enqueuing states you introduced?

Reply:
Yup, good catch, that was a work-in-progress comment. I've deleted it.